crawl-ref/source/dat/des/arrival/small.des

```diff
@@ -3412,3 +3412,126 @@ MAP
   ...........
    .........
 ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_battle_scene
+TAGS:    arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT:  float
+ITEM:    human skeleton / dwarf skeleton , \
+         goblin skeleton / gnoll skeleton / \
+         kobold skeleton / orc skeleton
+SHUFFLE: cx
+SHUFFLE: tvw
+FTILE:   .{det+ = floor_marble
```

Comment:
When you use `FTILE` be sure to also think about how the walls look and if you want custom walls, and be sure to set a `COLOUR` directive for console players!